index.html

```diff
@@ -263,7 +263,7 @@ <h1 class="features-title">How can you join?</h1>
           </p>
 
           <div class="wrapper">
-            <a class="btn-join" href="https://realdevsquad.com/join">
+              <a class="btn-join" href="javascript:void(0)">
```

Comment:
Why it inside div not button??
button has disabled attribute

let me know if I written something wrong?

Reply:
Yes your are right there should be button but already written code is handling it by anchor tag
Only requirment was that I need to replace value of href

Reply:
Okk @AdityaRaj2097, let @ajoykumardas12 approve your pr
then i will also approve.

Reply:
Sure I m working on @ajoykumardas12 changes

Reply:
@ajoykumardas12 can you tell me how u did attached image here when I m uploading it's showing my image link